apps/wallet/src/ui/app/hooks/useAmplitudeIdentity.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useEffect } from 'react';
+import Browser from 'webextension-polyfill';
+
+import { useActiveAccount } from './useActiveAccount';
+import useAppSelector from './useAppSelector';
+import { ampli } from '_src/shared/analytics/ampli';
+
+export function useAmplitudeIdentity() {
+	const activeAccount = useActiveAccount();
+	const { apiEnv, customRPC, activeOrigin } = useAppSelector((state) => state.app);
+	const activeNetwork = customRPC && apiEnv === 'customRPC' ? customRPC : apiEnv.toUpperCase();
+
+	useEffect(() => {
+		ampli.identify(undefined, {
```

Comment:
I think it would probably be helpful to add the current page info (e.g. href) as a user property, will think on this a little more tomorrow morning